server/src/controller/user.ts

```diff
@@ -0,0 +1,50 @@
+import pool from '../util/db.js';
+
+export const getUserData = (req:any,res:any)=>{
+
+    pool.query('SELECT * from users',(q_err:any,q_res:any)=>{
+        if(q_err){
+            console.log(q_err);
+            res.code = 404
+            res.json("some error occur",q_err);
+        }
+        res.json(q_res.rows);
+    })
+}
+
+export const deleteUser = (req:any,res:any)=>{
+    console.log(req.query.userid);
+    pool.query('DELETE FROM users WHERE user_id = $1',[req.query.userid],(q_err:any,q_res:any)=>{
```

Comment:
accept user id as a path param

Reply:
use the path param for userid in both deleteuser and update user